assets/src/pages/room/hooks/useToggle.tsx

```diff
@@ -0,0 +1,24 @@
+import { useCallback, useState } from "react";
+
+type UseToggleResult = [boolean, () => void];
+
+export const useToggle = (initialState = false): UseToggleResult => {
```

Comment:
remove this one, rename useCallbackToggle to useToggle and make callback optional